webactivity.py

```diff
@@ -530,6 +530,22 @@ def _key_press_cb(self, widget, event):
             elif key_name == 'w':
                 _logger.debug('keyboard: close tab')
                 self._tabbed_view.close_tab()
+            elif key_name == "Tab":
+                _logger.debug('keyboard: next tab')
+                current_index = self._tabbed_view.get_current_page()
+                if current_index == self._tabbed_view.get_n_pages() - 1:
+                    self._tabbed_view.set_current_page(0)
+                else:
+                    self._tabbed_view.set_current_page(current_index + 1)
+            elif event.get_state() & Gdk.ModifierType.SHIFT_MASK:
+                if key_name == "ISO_Left_Tab":
```

Comment:
No, there is no particular reason. Even Pippy had the same convention so I decided to use it here. Is there a problem?